packages/remix-server-runtime/cookies.ts

```diff
@@ -58,19 +58,22 @@ export interface Cookie {
   parse(
     cookieHeader: string | null,
     options?: CookieParseOptions
-  ): Promise<any>;
+  ): Promise<Value | null>;
 
   /**
    * Serializes the given value to a string and returns the `Set-Cookie`
    * header.
    */
-  serialize(value: any, options?: CookieSerializeOptions): Promise<string>;
+  serialize(
+    value: Value | null,
+    options?: CookieSerializeOptions
+  ): Promise<string>;
```

Comment:
@mikeybinnswebdesign As explained in https://stackoverflow.com/a/53573622/4001231, setting the cookie with an expired date in the past is the way [the spec](https://tools.ietf.org/search/rfc6265) defines how you should delete a cookie.

> Finally, to remove a cookie, the server returns a Set-Cookie header with an expiration date in the past.  The server will be successful in removing the cookie only if the Path and the Domain attribute in the Set-Cookie header match the values used when the cookie was created.

```suggestion
  serialize(value: Value, options?: CookieSerializeOptions): Promise<string>;
```